cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -274,6 +287,10 @@ public void testDeleteAll() throws Exception {
     assertResponseCode(403, deleteNamespace(NAME));
     Assert.assertTrue(nsLocation.exists());
     stopProgram(program);
+    // because adapter is started
+    assertResponseCode(403, deleteNamespace(NAME));
```

Comment:
This should not be 403. It should be 409. Not related to current PR. So change in a new PR?